deployments/ansible/roles/collector/defaults/main.yml

```diff
@@ -34,3 +34,42 @@ splunk_fluentd_config_source: ""
 
 agent_bundle_dir: /usr/lib/splunk-otel-collector/agent-bundle
 splunk_collectd_dir: /usr/lib/splunk-otel-collector/agent-bundle/run/collectd
+
+# default vars For Windows family
+
+splunk_api_url: https://api.{{splunk_realm}}.signalfx.com
+
+splunk_ingest_url:  https://ingest.{{splunk_realm}}.signalfx.com
+
+splunk_trace_url: "{{splunk_ingest_url}}/v2/trace"
+
+splunk_hec_url: "{{splunk_ingest_url}}/v1/log"
+
+splunk_hec_token: "{{splunk_access_token}}"
+
```

Comment:
These variables are not be Windows specific by definition. Can we remove them to avoid confusion? We can add them later for both linux and windows if needed

Reply:
For Linux these vars are getting fetched from templates and getting copied into splunk-otel-collector.conf. Let me try creating a separate file for Windows vars, if it works I will commit those changes.